addons/payment_paypal/models/payment.py

```diff
@@ -195,6 +191,20 @@ def _paypal_form_validate(self, data):
             'acquirer_reference': data.get('txn_id'),
             'paypal_txn_type': data.get('payment_type'),
         }
+        if not self.acquirer_id.paypal_pdt_token and not self.acquirer_id.paypal_seller_account and status in ['Completed', 'Processed', 'Pending']:
+            template = self.env.ref('payment_paypal.mail_template_paypal_invite_user_to_configure', False)
+            if template:
+                render_template = template.render({
+                    'acquirer': self.acquirer_id,
+                }, engine='ir.qweb')
+                mail_body = self.env['mail.thread']._replace_local_links(render_template)
+                mail_values = {
+                    'body_html': mail_body,
+                    'subject': _('Add your Paypal account to Odoo'),
+                    'email_to': self.acquirer_id.paypal_email_account,
```

Comment:
could you add an `email_from` odoobot here?
otherwise, the configuration email can potentially be sent by the portal user who does the first transaction and it's really weird O_o

after that we're good to go!

Reply:
Hello @dbo-odoo 

Changes are done with odoobot email
But I have one question, can we do it with the creator of acquirer?

Thanks

Reply:
wouldn't it be odoobot anyway since most providers are created from data files during installation of the payment module?

Reply:
@dbo-odoo 
I have tried it, in all cases (install from the terminal, button click install) I got bot user as creator